### DISCUSSION TITLE
Moonphases

### DISCUSSION THREAD

**Discussion by AndreasLMeg** [Alex-developer/allskyannotate#5] [Q&A]
Hello again,

I'm not 100% sure but I think the phases are not correct if white text color is used.
And, also not sure: is it the same for the southern part of the earth ?

![grafik](https://user-images.githubusercontent.com/85451655/147889830-0f7a822e-113b-4ecd-8716-f67e5885b621.png)

Andreas

**Comment by Alex-developer** [comment]
Well spotted, its n my list to invert the font just not got around to it yet

Cheers

Alex
